src/property.html

```diff
@@ -0,0 +1,1066 @@
+                  </div>
+                  <!-- hotel rating and info -->
+                  <div class="flex flex-col gap-1">
+                    <p class="font-semibold text-lg xl:text-xl text-color">
+                      Seaside Serenity Villa
+                    </p>
+                    <p
+                      class="font-medium text-sm xl:text-base text-color-addition-info"
+                    >
+                      Wake up to the soothing melody of waves. This beachfront
+                      villa offers...
+                      <a
+                        href="#"
+                        class="text-color text-underline-color-for-pricing-read-more"
+                      >
+                        Read More
+                      </a>
+                    </p>
+                  </div>
+                </div>
+                <!-- pricing -->
+                <div class="flex gap-4 items-center sm:gap-0 justify-between">
+                  <div>
+                    <p class="font-medium text-sm text-color-addition-info">
+                      Price
+                    </p>
+                    <p class="font-semibold text-sm sm:text-lg text-color">
+                      $1,250,000
+                    </p>
+                  </div>
+                  <div
+                    class="px-5 py-[14px] rounded-lg bg-for-second-btn font-medium text-[10px] sm:text-sm text-color xl:w-60 text-center"
+                  >
+                    View Property Details
+                  </div>
+                </div>
+              </div>
+              <div
+                class="p-5 sm:p-8 2xl:p-10 border-button rounded-xl gap-6 2xl:gap-8 md:grid hidden"
+              >
+                <img
+                  src="/public/assets/propertypage/hotel-features-section-02.png"
+                  alt="hotel"
+                  class="w-full"
+                />
+                <!-- location hotel -->
+                <div class="flex flex-col gap-4 2xl:gap-5 items-start">
+                  <div
+                    class="text-xs sm:text-sm 2xl:text-lg font-medium text-color border-button bg-for-logo-section px-3 py-1.5 rounded-3xl"
+                  >
+                    Urban Oasis - Life in the Heart of the City
+                  </div>
+                  <!-- hotel rating and info -->
+                  <div class="flex flex-col gap-1">
+                    <p class="font-semibold text-lg xl:text-xl text-color">
+                      Metropolitan Haven
+                    </p>
+                    <p
+                      class="font-medium text-sm xl:text-base text-color-addition-info"
+                    >
+                      Immerse yourself in the energy of the city. This modern
+                      apartment in the heart...
+                      <a
+                        href="#"
+                        class="text-color text-underline-color-for-pricing-read-more"
```

Comment:
shitty class names, also there's tailwind built in class to underline called underline

Reply:
fixed it